addons/web/static/tests/views/list_view_tests.js

```diff
@@ -666,6 +666,46 @@ QUnit.module("Views", (hooks) => {
         assert.containsNone(cpButtons[0], 'button[name="y"]');
     });
 
+    QUnit.test("list view: action button in controlPanel basic rendering", async function (assert) {
+        await makeView({
+            type: "list",
+            resModel: "foo",
+            serverData,
+            arch: `
+                <tree>
+                    <header>
+                        <button name="display" type="object" class="display" string="display" display="always"/>
+                        <button name="selection" type="object" class="selection" string="selection" display="selection"/>
```

Comment:
I would remove `display="selection"` as I would prefer people not to start defining that. What I want to see in archs:
`<button name="" type=""/>` like now
`<button name="" type="" display="always"/>`
but ideally never
`<button name="" type="" display="selection"/>` as it's the default